prompt2model_demo.py

```diff
@@ -181,8 +181,37 @@ def main():
             TaskType.TEXT_GENERATION, status["instruction"], status["examples"]
         )
         line_print("Retrieving dataset...")
+        line_print("Do you want to retrieve dataset automatically? (y/n)")
```

Comment:
I think this command line prompt needs to be made a bit more descriptive.

```suggestion
        line_print("Do you want to automatically select and transform the retrieved dataset? If no, you will be given the option to select the dataset to use from a shortlist of the best options (y/n)")
```